generators/app/templates/src/app/components/SearchBar/layout.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { reduxForm } from 'redux-form';
+
+import { SEARCH_FORM } from './constants';
+
+function SearchBar({ textButtonSearch, className, children }) {
+  return (
+    <div className={className}>
+      {children}
+      <button type="submit">
+        {textButtonSearch}
+      </button>
+    </div>
+  );
+}
+
+SearchBar.propTypes = {
+  textButtonSearch: PropTypes.string,
+  className: PropTypes.string
+};
+
+SearchBar.defaultProps = {
+  className: '',
+  textButtonSearch: 'Buscar'
+};
+
+export default reduxForm({
```

Comment:
I wouldn't couple this with reduxForm. Let's leave it so the user may not use reduxForm, and eventually if they want to, they can do `reduxForm({ form: MY_FORM })(SearchBarContainer)`.